### PR TITLE
chart: render args more safely

### PR DESCRIPTION
**Pull request by anbrsap**

### CODE REVIEW

**Review by alxsap — APPROVED** [review]

**Comment by alxsap** [comment]
Btw. this file is yaml (superset of json). The resulting resource in Kubernetes has been correct (without comma).

**Comment by anbrsap** [comment]
> Btw. this file is yaml (superset of json). The resulting resource in Kubernetes has been correct (without comma).

You are right. I should have read the [Flow Sequences section in the YAML spec](https://yaml.org/spec/1.1/current.html#id931285) carefully:

> The final entry may omit the “,” character.

That means this change doesn't fix anything, but just uses the alternative block sequence style.

File: charts/steward/templates/deployment-run-controller.yaml [hunk 1]
```diff
@@ -33,13 +33,12 @@ spec:
...
         {{- end }}
-        args: [
-        "-qps={{ .Values.runController.args.qps }}",
-        "-burst={{ .Values.runController.args.burst }}",
+        args:
+        - {{ printf "-qps=%d" ( .Values.runController.args.qps | int ) | quote }}
+        - {{ printf "-burst=%d" ( .Values.runController.args.burst | int ) | quote }}
         {{- if .Values.runController.args.logVerbosity }}
-        "-v", {{ .Values.runController.args.logVerbosity | quote }},
+        - {{ printf "-v=%d" ( .Values.runController.args.logVerbosity | int ) | quote }}
```
Comment (alxsap) [inline charts/steward/templates/deployment-run-controller.yaml:40]: I'm not sure if it works with one parameter `-v=<verbosity>`

Comment (anbrsap) [reply charts/steward/templates/deployment-run-controller.yaml:40]: It's mentioned like this in the [K8s logging conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions).

`klog.InitFlags()` is based on the `flag` package. `flag.Parse()` allows `-flag=value` (see [docs](https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax)).

So, I'm confident it works.

Comment (alxsap) [reply charts/steward/templates/deployment-run-controller.yaml:40]: Tested, works.

**Review by alxsap — APPROVED** [review]

**Pull request merged**